.github/workflows/ci-pip-tools.yml

```diff
@@ -37,6 +37,7 @@ jobs:
         run: |
           pip install --upgrade pip
           pip install -r dev-requirements.txt
+          pip install -e .
```

Comment:
Probably, but then we are not using the pinned versions in `dev-requirements.txt` and might as well delete `pip-tools` entirely.

Reply:
I'm leaving it how I had it for now, but it might be worth looking into how to best do this. Because I think if we try turn this into a package it won't use the pinned versions